### PR TITLE
cache daysInStudy and adjust queries for change

### LINKED ISSUE #680: Cache "days in study" on a participant document when ingesting data
Since the "days in study" of a participant don't change more than once a day, we can add an attribute for the days in study and cache the result of the current query we use to determine days in study so that we don't need to execute the expensive query when loading the participants page.

### PR DESCRIPTION
**Pull request by JackVCurtis**
closes #680

### CODE REVIEW

File: server/models/ParticipantsModel/index.js [hunk 1]
```diff
@@ -148,9 +120,9 @@ const allParticipantsQuery = (user, queryParams) => {
               },
             },
             {
-              participant: {
-                $in: searchParticipants,
-              },
+              $expr: {
```
Comment (Elkrival) [inline server/models/ParticipantsModel/index.js]: Document DB is missing support $expr 😢 

Screenshot 2024-03-12 at 12 18 03 PM

**Review by Elkrival — COMMENTED** [review]
I like it so far so good. We have to find a workour for `$expr`, that operator is unsupported by documentdb

File: server/models/ParticipantsModel/index.js (hunk 1 above)
Comment (JackVCurtis) [reply server/models/ParticipantsModel/index.js]: Gotcha, will look into a fix

Comment (JackVCurtis) [reply server/models/ParticipantsModel/index.js]: [18a4ef0](https://github.com/AMP-SCZ/dpdash/pull/676/commits/18a4ef0a8645d1c31d9e28a3eec8118b0fb3a720)

**Review by Elkrival — APPROVED** [review]
Oof, this looks so good @JackVCurtis 
🚀

**Pull request merged**